config/src/config/mod.rs

```diff
@@ -380,9 +380,9 @@ impl RootPath {
 mod test {
     use super::*;
 
-    const PUBLIC_FULL_NODE: &str = "src/config/test_data/public_full_node.config.yaml";
-    const VALIDATOR: &str = "src/config/test_data/validator.config.yaml";
-    const VALIDATOR_FULL_NODE: &str = "src/config/test_data/validator_full_node.config.yaml";
+    const PUBLIC_FULL_NODE: &str = "src/config/test_data/public_full_node.yaml";
+    const VALIDATOR: &str = "src/config/test_data/validator.yaml";
+    const VALIDATOR_FULL_NODE: &str = "src/config/test_data/validator_full_node.yaml";
```

Comment:
Seems like you're just focussing on the tests here, right? Do we have a plan for updating config-builder and the corresponding docker containers to follow the new naming convention? e.g., config-builder/src/main.rs still generates "*.config.toml" files :(

Reply:
well that is indeed problematic :D but not something we should do in this PR due to the tangled aspect of that code. Perhaps once we move pre-mainnet off config-builder we can follow up. Should make sure we sync with dev-net and testnet usage -- but CT should be self-service.